src/components/App/app-theme/app.typography.js

```diff
@@ -1,4 +1,4 @@
-export const typography = {
```

Comment:
Rename the file to `typography.styles.js, please, because it contains only styles

Reply:
Renamed!